subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/DefaultFileAccessTimeJournal.java

```diff
@@ -36,9 +38,12 @@
 
     public static final String CACHE_KEY = "journal-1";
     public static final String FILE_ACCESS_CACHE_NAME = "file-access";
+    public static final String META_PROPERTIES_FILE_NAME = "meta.properties";
```

Comment:
Should we call this file differently? Like file-access.properties?

Reply:
Yeah, I guess that would be a bit clearer and less generic. I'll change it.